lib/Doctrine/DBAL/Platforms/DB2Platform.php

```diff
@@ -899,4 +899,13 @@ protected function getReservedKeywordsClass()
     {
         return Keywords\DB2Keywords::class;
     }
+
+    public function getListTableCommentsSQL(string $table) : string
+    {
+        $table = $this->quoteStringLiteral($table);
```

Comment:
Please do not change the variable semantic once it's declared. The original meaning is the value itself, the modified meaning is the SQL expression representing the value. Something similar to the following would be preferred (including the nowdoc):

https://github.com/doctrine/dbal/blob/9853ad0c0ed2d5a3f12704c01c405fc820d06438/lib/Doctrine/DBAL/Platforms/MySqlPlatform.php#L376-L384

Reply:
:+1: done